Source/WebCore/css/parser/CSSSupportsParser.cpp

```diff
@@ -171,7 +172,12 @@ CSSSupportsParser::SupportsResult CSSSupportsParser::consumeSupportsSelectorFunc
 CSSSupportsParser::SupportsResult CSSSupportsParser::consumeSupportsFontFormatFunction(CSSParserTokenRange& range)
 {
     ASSERT(range.peek().type() == FunctionToken && range.peek().functionId() == CSSValueFontFormat);
-    auto format = CSSPropertyParserHelpers::consumeFontFormat(range, m_parser.context(), true);
+
+    auto state = CSS::PropertyParserState {
+        .context = m_parser.context(),
+    };
```

Comment:
Seems great to go vertical and named for non-trivial cases, but for the trivial cases like this it seems sufficiently clear like this:
```suggestion
    auto state = CSS::PropertyParserState { m_parser.context() };
```

Reply:
Totally agree.